### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
Hi @lurch, care to review this PR ?

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -25875,6 +25877,7 @@ substituded->substituted
 substitudes->substitutes
 substituding->substituting
 substitue->substitute
+substitued->substitute, substituted,
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:25880]: I guess you could also add `substitues->substitutes,` ?

Comment (lurch) [reply codespell_lib/data/dictionary.txt:25880]: And _perhaps_ it makes sense to reorder your suggestions to `substitued->substituted, substitute,` :shrug:

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: c9e99b8

**Pull request merged**